### ISSUE TITLE
There are no active source control providers. Recurring issue.

### ISSUE THREAD

**Issue by mdahiemstra** [Microsoft/vscode#56647]
- VSCode Version: 1.26.0 (4e9361845dc28659923a300945f84731393e210d)
- OS Version: Mac OS 10.13.6

Steps to Reproduce:

It just happens. Did not find a way to revert the situation, It just was broken. Did not install any plugins or some kind.

Does this issue occur when all extensions are disabled?: Yes

Hi,

I saw there are already allot of people complaining about their version control disappearing, And some of them even suggest a fix (which worked for me a while back) by adding the following lines to the user settings

`
"git.enabled": true,
 "git.path": "/usr/bin/git",
`

I am not sure, but after updating to the new version of vscode (1.26.0) the version control just shows "There are no active source control providers". Even with these settings in place.

P.S. Git and Mercurial both dont work (I have a hg extension enabled)

And yes of course I am in a version controlled project (with a .git dir).

What I already tried:
- Ran `sudo xcodebuild -license accept` to assure I have no pending licenses that need to be accepted because of the git integration of mac os x.
- Ran without extensions
- Disabled some extensions I assumed had something to do with git
- Google'd for this issue, only finding old confirmed fixes that don't work

Anyone else experiencing this? If you need any more info let me know.

**Comment by joaomoreno** [comment]
Interesting. It was working in 1.25 and stopped working in 1.26, correct?

Can you open the devtools and see whether there's any error there? <kbd>F1</kbd>, `Developer Tools`.

Anything else which doesn't work?

**Comment by mdahiemstra** [comment]
Hi, Yes that is correct. There are no errors in the console.
Everything else is fine.

**Comment by joaomoreno** [comment]
What about in <kbd>F1</kbd>, `Git: Show Git Output`?

**Comment by mdahiemstra** [comment]
Both are completely empty.

**Comment by mdahiemstra** [comment]
Wow, I just found that it IS an extension causing this. I disabled them one by one, and source control came back after disabling **vscode-spotify** (https://github.com/ShyykoSerhiy/vscode-spotify)

I'm guessing somehow this extension breaks loading components of the editor since the Running Extensions tab does not load anything until I disable this extension.

**Comment by joaomoreno** [comment]
Interesting, I can't repro that. Is it reproducible? If you enable it back does it still break git?

What if you reinstall it?

**Comment by mdahiemstra** [comment]
Hmm, this is very strange. 3 days ago I was able to reproduce it by enabling / disabling the vscode-spotify extension. But now it seems to be working fine.. I have no idea this extension might have updated itself or not, or the problem was something else. But then it is very strange that enabling/disabling it caused the other extensions not to load.

Also, when the other extensions did not load I also experienced some other issue, that the search in folder functionality gave arbitrary results. Weird. Seems to be fine now.
